templates/stamp/mongodb-ubuntu-install.sh

```diff
@@ -9,8 +9,12 @@ PACKAGE_NAME=mongodb-org
 REPLICA_SET_KEY_DATA=""
 REPLICA_SET_NAME=""
 REPLICA_SET_KEY_FILE="/etc/mongo-replicaset-key"
+
+#todo: make this configurable. lots of invocations to cleanup though
+#       grep -i -I -r 'mongo.*install.*script'
```

Comment:
yep. i'm just against hardcoding the same string in multiple spots. we should define it in once and properly plumb it in